scripts/qa_crowbarsetup.sh

```diff
@@ -3405,11 +3405,15 @@ function onadmin_proposal
     fi
     local proposal
     for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+        ceph monasca glance cinder neutron ; do
```

Comment:
Ah right. `oncontroller` asks for the nova one, this is unfortunate. I guest that we need to add a `oncluster`, that expect a cluster name as a parameter, to fix the issue : (

Thanks : )